worldengine/simulations/basic.py

```diff
@@ -33,36 +34,32 @@ def search(a, b, desired):
 
     all_land = width * height
     if ocean:
-        for y in range(0, height):
-            for x in range(0, width):
-                if ocean[y][x]:
-                    all_land -= 1
+        all_land -= numpy.count_nonzero(ocean)
     desired_land = all_land * land_percentage
     return search(0, 255, desired_land)
 
 
-def find_threshold_f(elevation, land_perc, ocean=None):
-    width = len(elevation[0])
-    height = len(elevation)
-    if ocean:
-        if (width != len(ocean[0])) or (height != len(ocean)):
+def find_threshold_f(map, land_perc, ocean=None, max=1000.0, mindist=0.005):
```

Comment:
can we get away from using map? rename back to elevation or use something generic like map_data or data?

map is a keyword: https://docs.python.org/2/library/functions.html#map